packages/perspective-jupyterlab/src/ts/view.ts

```diff
@@ -242,21 +242,28 @@ export class PerspectiveView extends DOMWidgetView {
         const table_options = msg.data["options"] || {};
 
         if (this.pWidget.client) {
-            // In client mode, retrieve the serialized data and the options
-            // passed by the user, and create a new table on the client end.
+            /**
+             * In client mode, retrieve the serialized data and the options
+             * passed by the user, and create a new table on the client end.
+             */
             const data = msg.data["data"];
             this.pWidget.load(data, table_options);
         } else {
             if (this.pWidget.server && msg.data["table_name"]) {
-                // Get a remote table handle, and load the remote table in
-                // the client.
+                /**
+                 * Get a remote table handle, and load the remote table in
+                 * the client for server mode Perspective.
+                 */
                 const table = this.perspective_client.open_table(msg.data["table_name"]);
                 this.pWidget.load(table, table_options);
-            } else if (msg.data["table_name"] && msg.data["view_name"]) {
-                // Get a remote view handle from the Jupyter kernel, and
-                // create a client-side table using the view handle.
+            } else if (msg.data["table_name"]) {
+                /**
+                 * Get a remote table handle from the Jupyter kernel, and
+                 * create a view on that handle to run Perspective in
+                 * distributed mode.
+                 */
```

Comment:
I can't (AFAIK) enforce this in our lint rules, but please use `//` exclusively for non-docstring comments.
Specifically, the `/** .. */` format has a specific [historical context of referring to a docstring](https://en.wikipedia.org/wiki/Javadoc), so this extra confusing to ex-java developers like myself!